### ISSUE TITLE
eventLimit for timeline view

### ISSUE THREAD

**Issue by jasonEXA** [fullcalendar/fullcalendar#4827]
Hi,
I am currently testing the scheduler library for use.
One possible issue I have encountered is with the eventLimit argument.
I have currently set eventLimit to true for all views of the calendar.

This works perfectly in agendaMonth and agendaWeek view.
However, for timelineWeek view,

```
customWeekView: {
              buttonText : 'Custom Week View',
              type: 'timeline',
              duration: { weeks: 1 },
              slotDuration: '24:00:00'
          }
```

eventLimit is not being applied.
I tried setting eventLimit to 2 in the customWeekView.  However, this does not get applied and all events get displayed as is.
jsBin output link - http://output.jsbin.com/dunojoqome/

One other UI issue I encountered (demonstrated in above jsBin link) is when there are multiple events, one of which extends over multiple days and the other is present for a single day for the same resource.  The one that extends over multiple days, overlaps the one on the single day.  There is no visual indication to the user if there is an event present on that single day.

E.g
![image](https://cloud.githubusercontent.com/assets/9958408/12788353/237558de-cabf-11e5-9a2d-c51909341e28.png)

Can this be confirmed if they are issues or just me mucking around? :)

**Comment by alizaidi606** [comment]
@arshaw when will this be available? I see this is not available in v5 either.

**Comment by steverob** [comment]
+1 for this. Any updates?

**Comment by ngthtung** [comment]
+1 for this. Any updates?

**Comment by Fogteen** [comment]
A +1 for this feature request (eventLimit support in timeline view).
Thanks @arshaw for your awesome work.

**Comment by arshaw** [comment]
This has been implemented in v5.7.0
blog post: https://fullcalendar.io/blog/2021/05/new-features-in-v570

However, some people might want timeline's +more links to render differently when the slot granularity is a day. This is not yet implemented. See this ticket:
https://github.com/fullcalendar/fullcalendar/issues/6305
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?
